README.md

```diff
@@ -33,6 +33,7 @@ Optional:
   - Linux >= 2.6.13
 - [gamin >= 0.0.21](http://www.gnome.org/~veillard/gamin)
 - [systemd >= 204](http://www.freedesktop.org/wiki/Software/systemd)
+- [dnspython or dnspython3](http://www.dnspython.org/)
```

Comment:
I wanted to, but I'm thinking the ability to query a text record has always been present, so I didn't want to specify a more recent version without verifying.